src/wrapper/simulation_object_updating_component.py

```diff
@@ -120,11 +138,33 @@ def _fetch_initial_simulation_objects(self):
             if len(node.getConnections()) >= 3
         ]
 
-        # signals
+        # other nodes
+        self._simulation_objects += list(
+            filter(
+                lambda x: x is not None,
+                (
+                    Node.from_simulation(node, self)
+                    for node in net.getNodes()
+                    if len(node.getConnections()) < 3
+                ),
+            )
+        )
+
+        # Edges
         self._simulation_objects += [
-            Signal.from_simulation(signal, self) for signal in net.getTrafficLights()
+            Edge.from_simulation(edge, self) for edge in net.getEdges()
         ]
 
+        # Tracks
+        for edge in (x for x in self.edges if x.track is None):
+            if edge.identifier.endswith("-re"):
+                identifier = edge.identifier.split("-re")[0]
+            else:
+                identifier = edge.identifier + "-re"
+
```

Comment:
Well, a Track consists of two SUMO-Edges (one for each direction). @Lietze requested, that we add the one-directional edges as well as the track (bi-directional). Yaramo, or more specifically the sumo-exporter, export the reverse-edges as the same id with a `-re` attached. Here we generate the appropriate reverse-edge-identifier (if we are currently looking at an id with `-re`, we remove it, otherwise we add it).